src/streams.h

```diff
@@ -531,19 +531,20 @@ class CBufferedFile
             readNow = nAvail;
         if (readNow == 0)
             return false;
-        size_t read = fread((void*)&vchBuf[pos], 1, readNow, src);
-        if (read == 0) {
+        size_t nBytes = fread((void*)&vchBuf[pos], 1, readNow, src);
+        if (nBytes == 0) {
```

Comment:
These changes are in this PR because we haven't cherry-picked bitcoin/bitcoin#8808.